pinot-server/src/main/resources/api/index.html

```diff
@@ -0,0 +1,105 @@
+<!DOCTYPE html>
```

Comment:
This looks like the stock swagger index file, if so, you should pull it from the webjar or document the changes from the stock version.